### PR TITLE
Fixes #14164: add back content counts to content view filter repos.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/14164

### CODE REVIEW

**Comment by stuarta** [comment]
Does this also address the issue in the downstream bugzilla that the counts are always zero, or is this just a reversal of the removal of the counts???

**Comment by waldenraines** [comment]
> Does this also address the issue in the downstream bugzilla that the counts are always zero, or is this just a reversal of the removal of the counts???

Yes.  It also fixes the counts.

File: app/views/katello/api/v2/content_view_filters/base.json.rabl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+node(:type) { |filter| filter.class::CONTENT_TYPE }
+attributes :inclusion
+
+child :content_view => :content_view do
+  extends 'katello/api/v2/content_views/show'
+end
+
+child :repositories => :repositories do
+  extends 'katello/api/v2/repositories/show'
```
Comment (jlsherrill) [inline app/views/katello/api/v2/content_view_filters/base.json.rabl]: I know this was using show before, but you can you change it to base?

Comment (jlsherrill) [inline app/views/katello/api/v2/content_view_filters/base.json.rabl]: repeat for content view above.

**Comment by waldenraines** [comment]
@jlsherrill updated.

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**